lua/neogen/templates/roxygen2.lua

```diff
@@ -0,0 +1,9 @@
+local i = require("neogen.types.template").item
+
+return {
+    { nil, "#' @title", { no_results = false, type = { "func" } } },
+    { nil, "#' @description", { no_results = false, type = { "func" } } },
+    { i.Parameter, "#' @param %s" },
+    { nil, "#' @return", { type = { "func" }, no_results = false } },
+    { nil, "#' @export", { type = { "func" }, no_results = false } }
```

Comment:
I suggest adding snippet default location:

take a look at https://github.com/danymat/neogen/blob/main/docs/adding-languages.md#default-generator, that would be something like:


```lua
...
 return {
     { nil, "#' @title $1", { no_results = false, type = { "func" } } }, -- notice the $1
 }
...
```

Reply:
Thank you, I have added this.